hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncScanSingleRegionRpcRetryingCaller.java

```diff
@@ -573,7 +574,8 @@ private void call() {
     resetController(controller, callTimeoutNs, priority);
     ScanRequest req = RequestConverter.buildScanRequest(scannerId, scan.getCaching(), false,
       nextCallSeq, scan.isScanMetricsEnabled(), false, scan.getLimit());
-    stub.scan(controller, req, resp -> onComplete(controller, resp));
+    Context context = Context.current();
```

Comment:
What is the purpose here? Wrap the callback in the span in the current context?

Reply:
In fact, this API has been added upstream, https://github.com/open-telemetry/opentelemetry-java/commit/cffbd3249c97d7d52ce19d55b509e9d7eb05ae26